src/nix_auto_follow/cli.py

```diff
@@ -71,9 +71,9 @@ def from_dict(cls, data: dict[str, Any]) -> "LockFile":
 
     def to_dict(self) -> dict[str, Any]:
         return {
+            "nodes": {key: value.to_dict() for key, value in self.nodes.items()},
```

Comment:
are the keys within the nodes sorted?

looks like the inputs are sorted too.

Reply:
I guess doesn't matter since we don't change the order of them.

Reply:
I think it's fine as is. I'll send another PR if I observe this happening.